LICENSE.md

```diff
@@ -0,0 +1,27 @@
+Copyright 2019-2021 by Felix Fontein <felix@fontein.de>.
```

Comment:
Q: Now that https://github.com/ansible-community/ansible-pygments/commit/342797fa934e233f5d38bc2fa6b3ef5db0746708 is in, should I also have a separate line? Should we just say "Ansible Community"? Should we add the lines just in that module?

Reply:
Where does `styles.py` come from, who contributed to it, and is it actually licensed under BSD 2-clause?

Reply:
I replaced my name with 'Ansible Community' in here. `styles.py` should have a notice as well, but I'm not sure what to put there since I don't know the answers for ^ @webknjaz